src/App.css

```diff
@@ -0,0 +1,9 @@
+.App {
+    text-align: center;
```

Comment:
Can you remove this line so that not all the text will be aligned to the center. It won't affect your WelcomeMessage component since it has it's own CSS that centers the text